mmdt/mmdt/urls.py

```diff
@@ -29,5 +13,4 @@
 if settings.DEBUG:
     import debug_toolbar
     urlpatterns.append(path('__debug__/', include(debug_toolbar.urls)))
-    urlpatterns += static(settings.MEDIA_URL,
-                          document_root=settings.MEDIA_ROOT)
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
In regards to the `mmdt/mmdt/urls.py file`, the same guidance applies for above comment